app/jobs/reports/monthly_account_reuse_report.rb

```diff
@@ -0,0 +1,133 @@
+require 'csv'
+
+module Reports
+    class MonthlyAccountReuseReport < BaseReport
```

Comment:
indent by 2 please!

Reply:
Sorry!  I forgot to run the linter on this. Fixed!